src/steps/vulnerability/converter.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  createIntegrationEntity,
+  Entity,
+  parseTimePropertyValue,
+} from '@jupiterone/integration-sdk-core';
+import { AquasecVulnerability } from '../../types';
+
+import { Entities } from '../constants';
+
+export function createVulnerabilityKey(name: string, image: string): string {
+  return `aquasec_trivy_vulnerability:${name},${image}`;
+}
```

Comment:
This key generation is purposefully done even if it results in duplication of CVE ids.

I did it this way because the returned data from the vulnerability endpoint has some [image-specific information](https://github.com/Creativice-Oy/graph-aquasecurity-trivy/blob/8fd2c4df03802d8b4fe225be7e153d320468bf8a/src/steps/vulnerability/converter.ts#L101-L106) which I thought was important to retain. 

An alternative would be to avoid the duplication by removing the image name from the key and removing the image-specific information. 

Or maybe just using `mappedRelationships` instead, as discussed.